Lib/test/test_listcomps.py

```diff
@@ -710,16 +710,19 @@ def test_multiple_comprehension_name_reuse(self):
         self._check_in_scopes(code, {"x": 2, "y": [2]}, ns={"x": 3}, scopes=["function", "module"])
 
     def test_name_collision_locals(self):
+        # GH-130809: The existence of a hidden fast from list comprehension should not cause
```

Comment:
I'm not a super strict 80-character person, but these two lines seem a bit longer than the rest of the code. Let's make it into 3 lines.

Reply:
Updated!